### DISCUSSION TITLE
Content Encryption | FixIt

### DISCUSSION THREAD

**Discussion by giscus** [hugo-fixit/FixIt#223] [General]
# Content Encryption | FixIt

Find out how to encrypt content in FixIt theme.

https://fixit.lruihao.cn/docs/content-management/encryption/

**Comment by Lruihao** [comment]
The current implementation is in **Node.js (TypeScript)**. For environments where Node.js performance is a bottleneck, community-maintained implementations in other languages are welcome:

| Language | Status    | Repository         |
| -------- | --------- | ------------------ |
| Node.js  | Available | [hugo-fixit/FixIt]  |
| Go       | Available | [fixit-encrypt.go] |

[hugo-fixit/FixIt]: https://github.com/hugo-fixit/FixIt/tree/main/packages/encrypt
[fixit-encrypt.go]: https://github.com/hugo-fixit/fixit-encrypt.go
